README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
-[] NaN
-[] Integers
-[] Arrays
-[] Null
+[X] Strings
+[X] Booleans
+[X] Undefined
+[X] NaN
+[X] Integers
+[X] Arrays
+[X] Null
```

Comment:
Close.  Not all.  

- JavaScript only has `Number`.  It does not define different types of numbers, like integers, short, long, floating-point etc.
- In JavaScript, an array is a special type of object. Therefore typeof [1,2,3,4] returns object.
- NaN (Not a Number) is not a datatype.